packages/server/src/cli.ts

```diff
@@ -139,8 +140,19 @@ if (isCli) {
 		}
 	}
 
-	logger.info(`\u001B[1mcode-server ${process.env.VERSION ? `v${process.env.VERSION}` : "development"}`);
+	if (isCli) {
+		let currentVersion = process.env.VERSION;
```

Comment:
This value is never reassigned. Use `const` instead?

```suggestion
		const currentVersion = process.env.VERSION;
```